### PR TITLE
Refactoring navigation, updates to headers, footer, and logo everywhere

### PR DESCRIPTION
**Pull request by boshevski**
Refactoring the navigation, logo, menu, and footer on all pages.

Using only one partial for the navigation navbar.html, and one partial for the footer footer.html, in all of the pages on the website.

Used the Storj Labs logo everywhere in navigation and footer. 

Other improvements related to the navigation include header updates on all pages, using the same design style for consistency across the website.

Removed the old sign-up page, and moved pages which were inside other directories for some reason.

Did many other improvements as I was going through the pages. 

The navigation and footer on the sign-up-node-operator page will have to be updated separately soon.

Improved responsiveness of the navigation and footer on all devices.

### CODE REVIEW

File: layouts/partials/partners/tabSelector.html [hunk 1]
```diff
@@ -1,26 +1,26 @@
...
-                        <li><a id="val3" href="#value3" data-toggle="tab" style="margin:15px;border-radius:4px;font-weight:600;padding-left:10px;padding-right:10px;padding-top:15px;padding-bottom:15px;">Platform Support</a></li>
-                        <li><a id="val4" href="#value4" data-toggle="tab" style="margin:15px;border-radius:4px;font-weight:600;padding-left:0px;padding-right:0px;padding-top:15px;padding-bottom:15px;">Joint Marketing</a></li>
-                        <li><a id="val5" href="#value5" data-toggle="tab" style="margin:15px;border-radius:4px;font-weight:600;padding-left:5px;padding-right:5px;padding-top:15px;padding-bottom:15px;">Early Access</a></li>
-                     </ul>
-                </div>
+    <div class="row">
+       <div class="navbar col-xs-12 col-lg-10 col-lg-offset-1" style="padding:0px;">
+           <ul class="nav nav-pills nav-justified text-center" style="padding:0px;">
+              <li class="active" ><a id="val1" href="#value1" data-toggle="tab" style="margin:15px;border-radius:4px;font-weight:600;padding-left:10px;padding-right:10px;padding-top:15px;padding-bottom:15px;">Revenue Sharing</a></li>
```
Comment (mbouzi) [inline layouts/partials/partners/tabSelector.html:17]: Maybe try to get rid of inline styling here

File: layouts/partials/storage-node-operator/heroNav.html [hunk 2]
```diff
@@ -37,20 +20,16 @@
...
-            <div class="col-sm-12">
-              <a href="#learnMore-sno" style="text-decoration: none">
-                <p class="mt-5 home-hero-cta">
-                  {{.Params.ctaText}}
-                </p>
-                <p style="font-size:12px;color:white" class="glyphicon glyphicon-chevron-down"></p>
-              </a>
-            </div>
-          </div>
+          <p class="sno-hero-sub-heading">{{.Params.subHeading}}</p>
```
Comment (mbouzi) [inline layouts/partials/storage-node-operator/heroNav.html:25]: Inline styles here as well

File: layouts/partials/partners/benefits.html [hunk 3]
```diff
@@ -1,33 +1,31 @@
...
+    <div class="row">
+      <div class="col-xs-12 col-sm-12 col-md-12 col-lg-12">
+        <h1 style="font-size:30px;text-align: center;">Why Your End-Users Will Love Us:</h1>
+      </div>
+    </div>
+
+    <div class="row">
+       {{ range where .Site.Pages ".Params.class" "1stSetBenefits"}}
+        <div class="col-md-4 col-lg-4 col-sm-12 col-xs-12  text-center" style="padding:50px;">
+          <img src="{{.Params.img}}" alt="Open-Source Software" class="icon-opensource">
```
Comment (mbouzi) [inline layouts/partials/partners/benefits.html:13]: inline styles here as well

**Review by mbouzi — DISMISSED** [review]

File: layouts/partials/partners/tabSelector.html (hunk 1 above)
Comment (mbouzi) [reply layouts/partials/partners/tabSelector.html:17]: sorry by here I meant on the page

**Comment by mbouzi** [comment]
Screen Shot 2020-04-15 at 2 04 44 PM
also missing a couple of edits that're in the current footer

File: layouts/partials/partners/tabSelector.html (hunk 1 above)
Comment (boshevski) [reply layouts/partials/partners/tabSelector.html:17]: There are inline styles on all of the pages, and those will have to be fixed when refactoring the whole website. This pull request is mostly for the navigation and footer. Also this page needs a bigger update.

File: layouts/partials/partners/benefits.html (hunk 3 above)
Comment (boshevski) [reply layouts/partials/partners/benefits.html:13]: same :)

**Review by mbouzi — APPROVED** [review]
👍

**Pull request merged**